alica_test_utility/src/Util.cpp

```diff
@@ -20,6 +20,19 @@ std::shared_ptr<BasicBehaviour> Util::getBasicBehaviour(alica::AlicaEngine* ae,
     return behaviour;
 }
 
+BasicPlan* Util::getBasicPlan(alica::AlicaEngine* ae, int64_t planId, int64_t configurationId)
+{
+    BasicPlan* plan = nullptr;
+    for (auto& planEntry : ae->getPlanPool().getAvailablePlans()) {
```

Comment:
the master plan is stored separately in the plan pool, so this method wont be able to get it